src/Sfa.Tl.ResultsAndCertification.Web/ViewModel/AdminDashboard/AdminComponentExamViewModel.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Sfa.Tl.ResultsAndCertification.Common.Enum;
```

Comment:
This file can be removed. It is not used anywhere. It was taken from my old branch sorry!